docs/admin/authentication.md

```diff
@@ -123,14 +123,14 @@ into apiserver start parameters.
     `tar xzf easy-rsa.tar.gz`
     `cd easy-rsa-master/easyrsa3`
     `./easyrsa init-pki`
-1.  Generate a CA. (--batch set automatic mode. --req-cn default CN to use.)
-    `./easyrsa --batch "--req-cn=${MASTER_IP}@date +%s" build-ca nopass`
-1.  Generate server certificate and key.
+2.  Generate a CA. (`--batch` set automatic mode. `--req-cn` default CN to use.)
```

Comment:
The numbered list is still not formatting right.  You might need a second line break before the first item.